src/genie/libs/parser/nxos/show_interface.py

```diff
@@ -3092,28 +3099,48 @@ class ShowRunningConfigInterfaceSchema(MetaParser):
                                   Optional('suppress_arp'): bool,
                                   }
                              },
+                        Optional('mtu'): int,
+                        Optional('ip_address'): str,
+                        Optional('vrf_member'): str,
                         }
                    },
               }
 
 
-# =================================================
-# Parser for 'show running-config interface <WORD>'
-# =================================================
+# ======================================================
+# Parser for 'show running-config interface {interface}'
+# ======================================================
 class ShowRunningConfigInterface(ShowRunningConfigInterfaceSchema):
-    """Parser for show running-config interface <WORD>"""
+    """Parser for
+        show running-config interface {interface},
+        show running-config | section ^interface,
+        show running-config interface
 
-    cli_command = 'show running-config interface {interface}'
+        The 2nd and 3rd cmd returns the same result. The reason to have them both is so that users can use either
 
-    def cli(self, interface, output=None):
+    """
 
+    cli_command = ['show running-config interface {interface}',
+                   'show running-config | section ^interface',
+                   'show running-config interface']
+
+    def cli(self, interface=None, output=None):
+        # Determine command
         if output is None:
-            out = self.device.execute(self.cli_command.format(interface=interface))
+            if interface:
+                cmd = self.cli_command[0].format(interface=interface)
+            elif 'section ^interface' in self.cli_command:
```

Comment:
This will never return `True`. I took a quick look and I'm not certain where (or if) the command is passed so that you could check this. If the produce the same results you can simply use one of the other in an `else`